__init__.py

```diff
@@ -80,36 +73,34 @@ def _wiki_page_summary(self, result, auto_suggest):
             auto_suggest (bool): True if auto suggest was used to get this
                                  result.
         """
-        lines = 2
-        summary = wiki.summary(result, lines, auto_suggest=auto_suggest)
-
-        if "==" in summary or len(summary) > 250:
-            # We hit the end of the article summary or hit a really long
-            # one.  Reduce to first line.
-            lines = 1
-            summary = wiki.summary(result, lines, auto_suggest=auto_suggest)
+        summary = wiki.summary(result, auto_suggest=auto_suggest)
 
         # Clean text to make it more speakable
-        return re.sub(r'\([^)]*\)|/[^/]*/', '', summary), lines
+        return re.sub(r'\([^)]*\)|/[^/]*/', '', summary).split('.')
```

Comment:
Actually this doesn't help when you have something like "... U.S. government..."

Should we use Chatterbox's [Wikipedia for Humans](https://github.com/HelloChatterbox/wikipedia_for_humans) package or Jarbas's chunking package [Quebra Frases](https://github.com/OpenJarbas/quebra_frases)?

Reply:
The chunking is basically the same as before, I think. will check the lines algorithm in wikipedia. wikipedia_for_humans is a good thing but is more of a complete overhaul since it will not provide the exact same results as the current backend and things like disambiguation and auto-suggest. I'd like to do this change but it's more of a complete rewrite...

Quebra phrases looks awesome. will try to add that.